src/core_modules/capture-core/components/Pages/NewEvent/DataEntry/epics/newEventDataEntry.epics.js

```diff
@@ -149,11 +148,12 @@ export const resetRecentlyAddedEventsWhenNewEventInDataEntryEpic = (action$: Inp
             return page === 'newEvent';
         }),
         filter((action) => {
-            // cancel if triggered by SELECTIONS_COMPLETENESS_CALCULATED and the underlying action is not SET_ORG_UNIT
+            // cancel if triggered by SELECTIONS_COMPLETENESS_CALCULATED and the underlying action is not SET_ORG_UNIT or SELECTIONS_FROM_URL_VALID
             const type = action.type;
             if (type === crossPageActionTypes.SELECTIONS_COMPLETENESS_CALCULATED) {
                 const triggeringActionType = action.payload && action.payload.triggeringActionType;
-                if (triggeringActionType !== lockedSelectorActionTypes.ORG_UNIT_ID_SET) {
+                if (![lockedSelectorActionTypes.SELECTIONS_FROM_URL_VALID, lockedSelectorActionTypes.ORG_UNIT_ID_SET]
+                    .includes(triggeringActionType)) {
```

Comment:
Arent we fixing this on a separate PR?

Reply:
Yes, comitted to master now (same commit). Should be ok I think.